doc/developer/webservices.rst

```diff
@@ -387,10 +387,13 @@ A standard xsd document that describe the layer.
 MapFish protocol
 ----------------
 
-URL: ``.../layers/<layer_id>/....``
+URL: ``.../layers/<layer_id>...``
 
 `Parameters and results, see the MapFish protocol <https://github.com/elemoine/papyrus/wiki/Protocol>`_.
 
+The only difference is that to Create/Update features (POST and PUT) you have to provide
+this header ``Content-Type: application/json``.
```

Comment:
Proposal
```
With GeoMapFish on Create/Update features (POST and PUT) you have to provide a correct ``Content-Type`` header sure as ``application/json``.
```

Reply:
@sbrunner thanks
@jwkaltz can I have your opinion or version ?
Thanks

Reply:
With GeoMapFish on Create/Update features (POST and PUT), you have to provide a correct ``Content-Type`` header, the value must be ``application/json``.

Reply:
Done thanks